source/nn/activations.lisp

```diff
@@ -284,3 +315,32 @@
   :assert-close ((x y) (every (~= 1e-6) x y))
   :in-place ((model) (= 2 (n-args `(100 100) model)))
   :kernel   ((model) (= 1 (n-kernels model))))
+
+
+(defun hardtanh-lisp (x &aux  (min_val -1.0)(max_val 1.0))(cond ((> x max_val) max_val)((< x min_val) min_val)(t x)))
```

Comment:
pay attention to horizontal spacing (c.f.: https://google.github.io/styleguide/lispguide.xml?showone=Horizontal_white_space#Horizontal_white_space)